src/components/NewCommentForm.tsx

```diff
@@ -1,6 +1,33 @@
-import React from 'react';
+import React, {
+  ChangeEvent,
+  FormEvent,
+} from 'react';
+import classNames from 'classnames';
+import { CommentData } from '../types/Comment';
+import { FieldErrors } from '../types/Errors';
+
+interface Props {
+  onCommentData: (
+    event: ChangeEvent<HTMLInputElement | HTMLTextAreaElement>,
+    newComment: CommentData
+  ) => void
```

Comment:
The onCommentData function should receive only one argument, the event. Remove the second argument 'newComment'.